config/sync/views.view.item_bulk_edit.yml

```diff
@@ -264,307 +271,8 @@ display:
           multi_type: separator
           multi_separator: ', '
           plugin_id: search_api
-        nid:
```

Comment:
why were some filters and sorts removed from the view?

Reply:
unintended -- it should only add the two actions... didn't realize this had changed since I switched to the latest code... I am going to have to discard these changes, import latest view from `develop` and add the two actions.